src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,14 @@
+$day-color: #eee;
+$day-size: 100px;
+$day-gap: 1px;
+$row-pad: 10px;
+$month-width: 7 * $day-size + 6 * $day-gap + 2 * $row-pad;
+$week-days: (
+  "start-day--mon": 0,
+  "start-day--tue": 1,
+  "start-day--wed": 2,
+  "start-day--thu": 3,
+  "start-day--fri": 4,
+  "start-day--sat": 5,
+  "start-day--sun": 6
+);
```

Comment:
I might have misread the task, I thought you only needed to create the mon, tue..etc parts, but maybe you're right, and it's a modifier from the start.